src/shared/components/case-list/case-list.component.ts

```diff
@@ -43,11 +43,11 @@ export class CaseListComponent {
   constructor(private browserService: BrowserService) { }
 
   public formatDate(date: Date): string {
-    return formatDate(date, 'dd MMM yyyy', 'en-GB');
+    return date ? formatDate(date, 'dd MMM yyyy', 'en-GB') : '-';
```

Comment:
Why "-"? I didn't see that mentioned on the ticket... 🤔

Reply:
The ticket describes a bug which is fixed by the introduction of a fallback for date fields.